features-json/input-datetime.json

```diff
@@ -227,10 +227,11 @@
       "9.9":"y"
     }
   },
-  "notes":"Older versions of Safari provide date-formatted text fields, but no real calendar widget.\r\n\r\nSome modified versions of the Android 4.x browser do have support for date/time fields.\r\n\r\nChrome and Opera also support the `datetime-local` type for both date and time, but not the `datetime` one.",
+  "notes":"Older versions of Safari provide date-formatted text fields, but no real calendar widget.\r\n\r\nChrome and Opera also support the `datetime-local` type for both date and time, but not the `datetime` one.",
```

Comment:
I had wanted to number the "Chrome and Opera" note too, but I'm not clear on whether it implies that no other browsers support `datetime-local` (if so, that seems like it deserves to be explicitly mentioned in and of itself).